lib/Core/ErrorState.cpp

```diff
@@ -132,11 +132,8 @@ void ErrorState::outputErrorBound(llvm::Instruction *inst, double bound) {
     }
   }
 
-  stream << errorVar << " == (";
+  stream << "\n";
   stream << PrettyExpressionBuilder::construct(e);
-  stream << ") && ";
```

Comment:
@Himeshi I noticed the bound constraints have been deleted. I agree it is a good thing to simplify the output, but it may still be useful to add an extra line containing a simple information about the specified bound, say "Absolute bound = ..." here.